src/api/github/getChangedStack.ts

```diff
@@ -21,13 +21,13 @@ export async function getChangedStack(
     // We can save on making extra calls to get GH client
     const octokit = client || (await getClient(ClientType.App, OWNER));
 
-    const { data } = await octokit.checks.listForRef({
+    const check_runs = await octokit.paginate(octokit.checks.listForRef, {
       owner: OWNER,
       repo,
       ref,
     });
```

Comment:
we can probably increase `per_page` to `100` (the maximum github allows) to reduce the pagination a bit